_pages/en/it-picture-long-term.md

```diff
@@ -119,56 +126,57 @@ As a result:
+EA committees and review boards at ESDC have aligned with the Policy and [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601) that enables transparency by working in the open and provides the [Mandatory Procedures for EA Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) that put users, open standards, open source software and cloud first as well as the [Mandatory Procedures on Application Programming Interfaces (API)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) that requires that services be available as APIs.
 Wherever possible code and projects are developed in the open and use of open standards and open source software is the norm.
 EA teams provide a loosely coupled architecture (reusable infrastructure artifacts and components) that empowers development and operations teams to meet business needs.
 
 ##### Continuous Delivery
 
 Production artifacts (code, configuration, documentation, ..) are kept using version control, in the open where possible.
 Changes trigger continuous integration (CI) tests (static, unit, sanity, security) that are run before changes can be accepted during code review.
-Continuous delivery (CD) pipelines are defined with a clear path to deploy to Production following tests, code review and a lightweight change approval process.
+Continuous delivery (CD) pipelines are defined with a clear path to deploy to Production following tests, code reviews and a lightweight change approval process.
 Small changes are frequently deployed to production.
-IITB monitors services, applications and infrastructure to check system health proactively as well as inform business decisions.
+IT monitors services, applications and infrastructure to check system health proactively as well as inform business decisions.
 
 Teams work using the agile methodology to gather and implement customer feedback, working in small batches (iteration or sprint).
 Where possible work is being done in the open make the code, data and information available to other teams and interested parties.
 
 ##### Internal IT services
 
-IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
-IITB is responsible for providing the automated services required to run a secure and agile business.
-This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions.
+IT is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
```

Comment:
That should be IITB. Internal IT services would remain in a corporate IT branch.

Reply:
There is currently no mention of IITB in the long term vision.  It's referred to only as IT or IT services.